control-plane/tests/test_phase58_1_doctor_contract.py

```
@@ -222,6 +222,84 @@ def test_doctor_contract_wazuh_explanation_reflects_tracked_sources(self) -> Non
             "Wazuh ingest prerequisites are configured",
         )
 
+    def test_doctor_contract_explains_required_broken_families_with_safe_next_steps(
+        self,
+    ) -> None:
+        payload = self._build_doctor_payload(
+            config_overrides={
+                "host": "0.0.0.0",
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)